woocommerce/woocommerce-android
# Hack week/add device info in app logs

This PR appends device information into the app logs. Whenever the user copies or shares the app logs. We append the device information to it.

### Why?
p7cLQ7-1H5-p2#comment-3162 has more info about why this is needed.

### Description
This PR appends device information into the app logs. Whenever the user copies or shares the app logs. We append the device information to it. This information would be helpful when testing or filing bug reports.

In this PR, we append the following device information

- Device Model and Name
- Device Operating System
- Device Language



### Testing instructions

1. Navigate to the `settings` screen from the `MyStore` screen by clicking the icon on the top right corner.
2. In the `settings` screen, click on `Help & Support` option
3. In `Help & Support` screen click on the `Application log` option
4. Click on the `Copy` icon in the toolbar and paste the copied item into any of the file and verify that device info has been appended at the end of the log
5. Click on the `Share` icon in the toolbar and share it into any of the file and verify that device info has been appended at the end of the log


### Images/gif
